web-console/monitoring/alertmanager-config/alertmanager-config.md

```diff
@@ -0,0 +1,32 @@
+# Alertmanager Configuration
+
+![alerts list view](img/alerts-list.png)
+- To consolidate all admin-level alerting functions together, `Alerts` and `Silences` pages could be combined onto the same page as tabs with the `Configuration` and `YAMl` pages
+
+![alerting configuation tab](img/config-empty.png)
+- The `Configuration` tab would show some basic grouping information from the default route section of the YAML and a list of receivers for the cluster.
+
+![configuration tab with create menu open showing receiver types](img/config-dropdown.png)
+- Clicking the “create” dropdown would enable users to create a receiver based on an integration type.
+- We could provide the entire list of integration types here, or just a subset as time allows.
+- Users would only be able to create receivers with a single type of integration. More complex receivers could be created from the `YAML` tab.
+
+![create receiver page](img/create-default-receiver.png)
+- The create receiver form would expose basic required information for the given integration type.
+- If there is no information in the `global` section of the config YAML for a given receiver type, possible global fields are added to the form with the option to write them to the `global` secton. If the box is not checked, they are simply added to the receiver itself.
+- Optional fields appear if the user clicks the `Show advanced configuration` button.
+- If no receivers have been created, no other fields are available as this receiver must become the default receiver for alerts.
+
+![create reciever page with routing labels section](img/create-receiver-long.png)
+- Once a default receiver exists, subsequent receiver creation forms will have a `Routing Labels` section that enables users to create a very basic routing rule for the receiver at the same time.
```

Comment:
What does the Regular expression checkbox do to the row?

Reply:
That means that the "value" field will be interpreted as a regular expression. It _might_ be possible to detect that automatically, but I think there's also value in being explicit here